src/VideoIntelligence/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "google/cloud-videointelligence",
+    "description": "Cloud Video Intelligence Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "require": {
+        "ext-grpc": "*",
+        "google/cloud-core": "^1.0",
```

Comment:
@jdpedrie This seems to be carried over from the other GAPIC only clients, do you know why core is required?